app/src/main/java/com/example/party_player/MainActivity.kt

```diff
@@ -28,16 +27,15 @@ class MainActivity : AppCompatActivity() {
 
     private val CLIENT_ID = "3d04daf658674a5c97e487250a7c49ef"
     private val redirectUri = "com.example.party-player://callback"
-    val AUTH_TOKEN_REQUEST_CODE = 0x10
-    val AUTH_CODE_REQUEST_CODE = 0x11
-    lateinit var spotifyAppRemote: SpotifyAppRemote
+    private val AUTH_TOKEN_REQUEST_CODE = 0x10
+    private val AUTH_CODE_REQUEST_CODE = 0x11
```

Comment:
I changed a lot of the variables to private as seen in the kotlin coding conventions website. 

Reference: https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules

Reply:
I agree... and good citation.